File: All Codes/URI_1049.c
#include <stdio.h>
int main()
{
    char a[15];
    char b[15];
    char c[15];
    scanf("%s",a);
    scanf("%s",b);
    scanf("%s",c);

    if (a[0] == 'v' && b[0] == 'a' && c[0] == 'c')printf("aguia\n");
    if (a[0] == 'v' && b[0] == 'a' && c[0] == 'o')printf("pomba\n");
    if (a[0] == 'v' && b[0] == 'm' && c[0] == 'o')printf("homem\n");
    if (a[0] == 'v' && b[0] == 'm' && c[0] == 'h')printf("vaca\n");
    if (a[0] == 'i' && b[0] == 'i' && c[2] == 'm')printf("pulga\n");
    if (a[0] == 'i' && b[0] == 'i' && c[2] == 'r')printf("lagarta\n");
    if (a[0] == 'i' && b[0] == 'a' && c[0] == 'h')printf("sanguessuga\n");
    if (a[0] == 'i' && b[0] == 'a' && c[0] == 'o')printf("minhoca\n");

    return 0;
}

File: All Codes/URI_1049.py
animal = {
    'aguia': ['vertebrado', 'ave', 'carnivoro'],
    'pomba': ['vertebrado', 'ave', 'onivoro'],
    'homem': ['vertebrado', 'mamifero', 'onivoro'],
    'vaca': ['vertebrado', 'mamifero', 'herbivoro'],
    'pulga': ['invertebrado', 'inseto', 'hematofago'],
    'lagarta': ['invertebrado', 'inseto', 'herbivoro'],
    'sanguessuga': ['invertebrado', 'anelideo', 'hematofago'],
    'minhoca': ['invertebrado', 'anelideo', 'onivoro']
}

def get_key(val):
    for key, value in animal.items():
        if val == value:
            return key

l = []
l.append(input())
l.append(input())
l.append(input())

print(get_key(l))


